### PR TITLE
Implement generic support for IRCv3 standard replies.

### PR DESCRIPTION
**Pull request by SadieCat**
Not much is really done with them currently other than printing them in a user-readable way but later specs may require special handling.

https://ircv3.net/specs/extensions/standard-replies

### CODE REVIEW

File: src/common/proto-irc.c [hunk 1]
```diff
@@ -460,6 +460,18 @@ channel_date (session *sess, char *chan, char *timestr,
 								  tags_data->timestamp);
 }
 
+static int
+trailing_index(char *word_eol[])
+{
+	int index;
```
Comment (TingPing) [inline src/common/proto-irc.c:466]: Avoid the name `index` since its in the stdlib.

File: src/common/proto-irc.c [hunk 2]
```diff
@@ -460,6 +460,18 @@ channel_date (session *sess, char *chan, char *timestr,
 								  tags_data->timestamp);
 }
 
+static int
+trailing_index(char *word_eol[])
```
Comment (TingPing) [inline src/common/proto-irc.c:464]:
```suggestion
trailing_index(const char *word_eol[])
```

**Comment by TingPing** [comment]
Looks good, I'll merge after minor nits.

**Comment by SadieCat** [comment]
Should be resolved now?

**Pull request merged**